README.md

```diff
@@ -42,6 +42,15 @@ This SDK attempts to follow [semver](https://semver.org/) in a best-effort basis
 
 The contents of previous release can be found on the [branch v1](https://github.com/auth0/react-native-auth0/tree/v1).
 
+### Platform compatibility
+
+The following shows platform minimums for running projects with this SDK:
+
+| Platform | Minimum version |
+| -------- | :-------------: |
+| iOS      |      13.0       |
+| Android  |       ???       |
+
```

Comment:
@poovamraj I think this table would be useful, what do you think?

If you agree, what should we say is the minimum for Android?